ee/controllers/phdeployment_controller.go

```diff
@@ -304,6 +297,18 @@ func (r *PhDeploymentReconciler) updateStatus(ctx context.Context, phDeployment
 		return nil
 	}
 
+	for _, c := range deployment.Status.Conditions {
+		if c.Type == v1.DeploymentReplicaFailure && c.Status == corev1.ConditionTrue && c.Reason == "FailedCreate" {
+			// create replicaset failed, because request exceeds quota and is denied by admission webhook
+			phDeployment.Status.Phase = primehubv1alpha1.DeploymentDeploying
+			phDeployment.Status.Messsage = c.Message
```

Comment:
About the message
![image](https://user-images.githubusercontent.com/860633/79737896-97660000-832e-11ea-9fe4-bb8f5aa2ca1f.png)

There is a story in jupyterhub
https://app.clubhouse.io/infuseai/story/5367/bug-update-the-error-message-when-there-is-not-enough-quota
https://gitlab.com/infuseai/primehub/-/merge_requests/740/diffs

Could we show the same message in model deployment?